subcommands/config/remote.go

```diff
@@ -68,16 +72,87 @@ func cmd_remote_config(ctx *appcontext.AppContext, args []string) error {
 		}
 		return ctx.Config.Save()
 
+	case "check":
+		usage := "usage: plakar remote check <name>"
+		if len(args) != 1 {
+			return fmt.Errorf(usage)
+		}
+		name := args[0]
+		if !ctx.Config.HasRemote(name) {
+			return fmt.Errorf("kloset %q does not exists", name)
+		}
+		_, err := storage.New(ctx.GetInner(), ctx.Config.Remotes[name])
+		if err != nil {
+			return err
+		}
+		return nil
```

Comment:
this looks wrong to me; this is not a storage config but an importer and/or exporter one, so we should try to instantiate that.  There's no guaranteed that a remote (which we might rename connector?) is also a storage; take ftp for instance: we can backup/restore it but we can't currently have a repository over bare ftp.  (or think of the work the interns are doing, we're not going to store a kloset in, say, notion or icloud photo)